crypt4gh/__main__.py

```diff
@@ -108,9 +104,8 @@ def run(args):
                 if os.path.isfile(k):
                     yn = input(f'{k} already exists. Do you want to overwrite it? (y/n) ')
                     if yn != 'y':
-                        print('Ok. Fair enough. Exiting.')
-                        #sys.exit(0)
-                        return
+                        LOG.error('Ok. Fair enough. Exiting.')
```

Comment:
Same as above. Keep the `print` statement, it is there on purpose.

Reply:
No problem.

Reply:
@silverdaz Logging is more amenable to plug into other log-collection subsystems... plus the `LOG.error` should show in the screen as the regular print() does. So I'm also curious on why you want to keep `print()`?:

https://fangpenlin.com/posts/2012/08/26/good-logging-practice-in-python/

See "print is not a good idea" :)

Reply:
The logger does not output anything if it is not configured to do so.
I use for example:
```version: 1
root:
  level: NOTSET
  handlers: [noHandler]

loggers:
  cryptography:
    level: DEBUG
    handlers: [console]
  crypt4gh:
    level: DEBUG
    handlers: [console]
    propagate: True


handlers:
  noHandler:
    class: logging.NullHandler
    level: NOTSET
  console:
    class: logging.StreamHandler
    formatter: simple
    stream: ext://sys.stderr

formatters:
  simple:
    format: '[{name:^10}][{levelname:^6}] (L{lineno}) {message}'
    style: '{'
```
which indeed prints to `stderr`.
However, I want some outputs to be forced to `stderr`.
I'm fully aware that `print` is not good for debugging, but this here is a different context.

In case the user wants to _also_ turn those outputs off, he can use `2>/dev/null` at the command line